source/process/release-scorecard-definitions.rst

```diff
@@ -0,0 +1,189 @@
+|                                        | customers, CSMs, Support or Product.    |
+|                                        | E.g. dot releases from customer reports,|
+|                                        | undocumented breaking changes.          |
++----------------------------------------+-----------------------------------------+
+| Number of setbacks to users/staff      | Manual count of negative reactions      |
+| resulting in a negative reaction       | reported by CSMs, Support or Product.   |
+|                                        | E.g. features that got pushed.          |
++----------------------------------------+-----------------------------------------+
+| Number of measurable actions from each | Count of completed and reported items   |
+| release retrospective driven from root | from Release Retrospective document.    |
+| causes & completed and reported to R&D |                                         |
++----------------------------------------+-----------------------------------------+
+| Number of deadlines missed in release  | Manual count of deadlines not met from  |
+| checklist                              | items posted to Release Checklist.      |
++----------------------------------------+-----------------------------------------+
+
+Release Heartbeat
+=================
+
++-----------------------------------------+--------------------------------------------------------------------------------+
+| Metric                                  | How to Measure                                                                 |
++=========================================+================================================================================+
+| Time updated to RC1 (PST)               | Check Release Discussion channel history                                       |
+|                                         | for date/time RC1 was cut.                                                     |
++-----------------------------------------+--------------------------------------------------------------------------------+
+| How many RCs cut                        | Check Release Discussion channel history                                       |
+|                                         | for how many RCs were cut for that release.                                    |
++-----------------------------------------+--------------------------------------------------------------------------------+
+| Number of days between when final RC    | Check Release Discussion channel for post with official release build.         |
+| is cut and the release date             | Oxygen = 16th - Day Final RC is cut                                            |
++-----------------------------------------+--------------------------------------------------------------------------------+
+| Community + customer bugs reported      | With a new or existing Jira filter, with:                                      |
```

Comment:
As you start working through the release scorecard, propose including a link to the Jira filter to use here as well (and in the release scorecard spreadsheet)